mcumgr-core/src/main/java/io/runtime/mcumgr/managers/FsManager.java

```diff
@@ -214,37 +303,21 @@ public synchronized void upload(@NotNull String name, @NotNull byte[] data,
         sendNext(0);
     }
 
-    //******************************************************************
-    // File Upload / Download
-    //******************************************************************
-
-    // Upload / Download states
-    public final static int STATE_NONE = 0;
-    public final static int STATE_UPLOADING = 1;
-    public final static int STATE_DOWNLOADING = 2;
-    public final static int STATE_PAUSED = 3;
-
-    // Upload / Download variables
-    private int mTransferState = STATE_NONE;
-    private String mFileName = null;
-    private int mOffset = 0;
-    private byte[] mFileData;
-    private FileUploadCallback mUploadCallback;
-    private FileDownloadCallback mDownloadCallback;
-
     /**
      * Get the current upload state ({@link #STATE_NONE},
      * {@link #STATE_UPLOADING}, {@link #STATE_DOWNLOADING}, {@link #STATE_PAUSED}).
      *
      * @return The current upload state.
      */
+    @Deprecated
     public synchronized int getState() {
         return mTransferState;
     }
 
     /**
      * Cancel an undergoing file transfer. Does nothing if no transfer is in progress.
      */
+    @Deprecated
```

Comment:
How about adding `@deprecated` in comment with a suggestion what to use instead?